lib/platform/bitbucket-server/index.ts

```diff
@@ -385,28 +384,21 @@ export async function findPr({
 }
 
 // Returns the Pull Request for a branch. Null if not exists.
-export async function getBranchPr(
-  branchName: string,
-  refreshCache?: boolean
-): Promise<BbsPr | null> {
+export async function getBranchPr(branchName: string): Promise<BbsPr | null> {
   logger.debug(`getBranchPr(${branchName})`);
   const existingPr = await findPr({
     branchName,
     state: PR_STATE_OPEN,
   });
-  return existingPr ? getPr(existingPr.number, refreshCache) : null;
+  return existingPr ? getPr(existingPr.number) : null;
 }
 
-export async function commitFiles(
-  commitFilesConfig: CommitFilesConfig
-): Promise<string | null> {
-  const commit = git.commitFiles(commitFilesConfig);
-
+// istanbul ignore next
+export async function refreshPr(number: number): Promise<void> {
   // wait for pr change propagation
   await delay(1000);
```

Comment:
I now know a way to remove the delay, but we need to make additional request to force the pr update.

On bitbucket server <7 we need to call the merge api and on >=7 we need to call changes api.
I've not yet tested it, but it maybe a better solution instead of waiting one second.

@rarkins Should we try this?

ref: jenkinsci/bitbucket-branch-source-plugin#294

Reply:
I don't mind a second if it's reliable (we don't rebase *all* that often).